Pod/Classes/CloudFivePush.m

```diff
@@ -137,42 +143,13 @@ - (void)application:(UIApplication *)application didReceiveRemoteNotification:(N
         [[UIApplication sharedApplication] setApplicationIconBadgeNumber:0];
     }
 
-    // If we're in the foreground and the app dev didn't implement the receiveRemoteNotificaiton method,
-    // We show an alert by default so the notification isn't swallowed.
-    BOOL appImpl = [[CloudFivePush sharedInstance] appDidImplementSelector:@selector(application:didReceiveRemoteNotification:)];
-    if (appState == UIApplicationStateActive && !appImpl) {
-        [[CloudFivePush sharedInstance] handleForegroundNotification:userInfo];
```

Comment:
Can you log a message/warning here saying a push was received but `didReceiveRemoteNotification` was not implemented?